front/js/shared/tracking.es6

```diff
@@ -32,9 +32,16 @@
   };
 
   function setUpListeners() {
-    mediator.subscribe('onOneClickBuy', () => {
+    mediator.subscribe('onOneClickBuy', (_, phone) => {
       reachGoal('CMN_BUY_SEND');
       reachGoal('FAST_BUY_SEND');
+      // @todo #977:120m  Create some lazy() wrapper for counters.
+      //  `lazy(my_counter).reachGoal()` should asynchronous delay reaching
+      //  until my_counter object won't be fully loaded.
+
+      // @todo #977:30m  Create mock class for carrotquest.
+      //  Fill it's methods with console logs.
```

Comment:
What is this for? Will it be used for testing?

Reply:
@artemiy312 , y, for testing. I'll append note about relation to test mech creating https://github.com/fidals/shopelectro/issues/978